ocp-templates/ocp-config/component-environment/component-template.yml

```diff
@@ -0,0 +1,146 @@
+apiVersion: v1

```

Comment:
I would say that this folder should not be in `ods-core` - I would expect it somewhere in `ods-quickstarters/common`